packages/commercetools/api-client/src/helpers/i18n/index.ts

```diff
@@ -0,0 +1,5 @@
+const getCurrency = () => 'USD'
```

Comment:
Imho it's better to keep it from config passed into `setup` then in the app you can set reference to currency and locale fields. It's also better to keep all configuration in one place